app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = list()
+    for person in people:
+        name = person["name"]
+        age = person["age"]
+        new_person = Person(name, age)
+        person_list.append(new_person)
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife = \
+                Person.people[person["wife"]]
```

Comment:
don't use `\`

Reply:
i can`t make my code work without reverse slash
Pycharm puts it automatically when I move the code line